PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+lorem ipsum lorem ipsum lorem ipsum lorem
+
+## Features/Fixes/Patches/Changes list:
+
+Please check the [CHANGLOG.md](https://github.com/boldlink/terraform-aws-dynamodb/blob/afea2f40ded4aaa22d5765b58ada380807fc1b82/CHANGELOG.md#)
```

Comment:
Fix typo on the CHANGELOG, the link on the template shouldn't have a hardcoded commit, instead, it should be the branch name or the link to the version you are releasing.